### PR TITLE
EDU-955: Change SSO mentions from Azure AD to MS Entra ID

### PR DESCRIPTION
**Pull request by fairlydurable**
Simple swap-outs

### CODE REVIEW

File: docs/production-deployment/cloud/saml.mdx [hunk 1]
```diff
@@ -85,7 +85,7 @@ To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
    - In the **SAML Certificates** section of the **SAML-based Sign-on** page, select the download link for **Certificate (Base64)**.
    - In the **Set up _APPLICATION_NAME_** section of the **SAML-based Sign-on** page, copy the value of **Login URL**.
 
-To finish setting up Azure AD as your SAML IdP, see [Finish SAML configuration](#finish-saml-configuration).
+To finish setting up Manage Microsoft Entra ID as your SAML IdP, see [Finish SAML configuration](#finish-saml-configuration).
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/saml.mdx:88]:
```suggestion
To finish setting up Microsoft Entra ID as your SAML IdP, see [Finish SAML configuration](#finish-saml-configuration).
```

File: docs/production-deployment/cloud/saml.mdx [hunk 2]
```diff
@@ -31,21 +31,21 @@ For more information, contact your account manager.
...
-To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
+To use Entra ID as your SAML IdP, create an Microsoft Entra ID Enterprise application.
 
-1. Sign in to the [Microsoft Azure AD portal](https://portal.azure.com/).
-1. On the home page, under **Manage Azure Active Directory**, select **View**.
+1. Sign in to the [Microsoft Entra ID](https://portal.azure.com/).
+1. On the home page, under **Manage Microsoft Entra ID**, select **View**.
 1. On the **Overview** page near the top, select **Add > Enterprise application**.
-1. On the **Browse Azure AD Gallery** page near the top, select **Create your own application**.
+1. On the **Browse Manage Microsoft Entra ID Gallery** page near the top, select **Create your own application**.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/saml.mdx:48]:
```suggestion
1. On the **Browse Microsoft Entra ID Gallery** page near the top, select **Create your own application**.
```

**Comment by jsundai** [comment]
LGTM but could you add context to the ticket like the jira (w slack convo etc) so I have more background.

**Review by jsundai — APPROVED** [review]

**Comment by fairlydurable** [comment]
Hi @jsundai Thanks for the review. The JIRA ticket is in the title of the PR.

**Pull request merged**